test/integration_test/common_test.go

```diff
@@ -180,3 +183,11 @@ func verifyScheduledNode(t *testing.T, appNode node.Node, volumes []string) {
 	logrus.Infof("Scores: %v", scores)
 	require.Equal(t, highScore, scores[appNode.Name], "Scheduled node does not have the highest score")
 }
+
+func init() {
+	flag.IntVar(&snapshotScaleCount,
+		"snapshot-scale-count",
+		10,
+		"Number of volumes to use for scale snapshot test")
```

Comment:
nit: this isn't just Number of volumes but rather the scale factor by which apps and volumes both get scaled.

Reply:
Yes, but the purpose is to test the scaling of volumes and snapshots, which is what it is referring to.